bundles/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/handler/MaxCubeBridgeHandler.java

```diff
@@ -842,11 +834,19 @@ private void setProperties(HMessage message) {
             for (Map.Entry<String, String> entry : properties.entrySet()) {
```

Comment:
Why are the properties not shown in PaperUI?

Reply:
as this was the very first binding ported to OH2... no such thing in PaperUI at that time...
now it is... so I can indeed remove that bit of code